.github/workflows/checks.yml

```diff
@@ -0,0 +1,9 @@
+name: Tests and Checks
```

Comment:
Can you merge checks.yml & test.yml? it's usually one file in most repos

Reply:
yea